setup.py

```diff
@@ -11,7 +11,7 @@
 install_requires = [
     "attrs<19",
     "psycopg2<2.8",
-    "click<7",
+    "click<=7.1.2",
```

Comment:
Generally, there are no restrictions on `click` features, and it might be better for future-compatibility to change the constraint to something like "click>=6"